### DISCUSSION TITLE
CTT Subsciption Basic - Keepalive is one publish interval to late.

### DISCUSSION THREAD

**Discussion by johannwesely** [eclipse-milo/milo#1302] [General]
Hi,
I have the following CTT Issues :
![image](https://github.com/user-attachments/assets/5886dd90-8190-4ce7-b5de-cbb431e151bd)
Have you an Idea what possibly could go Wrong ? Because i dont know where to start here.
Kind Regards
Johann Wesely

**Comment by kevinherron** [comment]
It looks to me like these tests and the server are off by one publishing interval in their expectations about when a keep alive response is to be returned. There's nothing you need to do about this, I'll have to open a Mantis issue in the OPC Foundation tracker to discuss this.

**Comment by kevinherron** [comment]
Turns out there was an errata issued that modifies the Subscription behavior. It's an easy fix.

https://github.com/eclipse/milo/issues/1304

**Reply by johannwesely** [reply]
Thanks a lot for your effort!
